package-lock.json

```diff
@@ -4364,7 +4364,7 @@
     },
     "browserify-aes": {
       "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
```

Comment:
This is changed to `http`. You might need to remove `node_modules` and reinstall modules to prevent it from happening.